### DISCUSSION TITLE
Hero 13 labs bug, unable to create new presets

### DISCUSSION THREAD

**Discussion by MtnXfreeride** [gopro/labs#1064] [General]
As seen in the video here. None of my hero 13s can create new video profiles after switching to labs firmware:
https://youtu.be/-1FjfsH2gY0?si=fgpsdrdfB-SlSk15

**Comment by dnewman-gpsw** [comment]
I have now confirmed this. As Labs doesn't modify the UI, so this is a bit of a surprise. All the current internal builds for the next Labs release don't seem have this error. So that fix is coming soon.

If you stay on Labs you can add Presets the Labs way (bypasses the Camera UI.)  You can use https://gopro.github.io/labs/control/custom/

or the command 

```mV$PRES="x,name"```

mV - sets the type, mode Video, or mT for mode Timelapse etc.

x - icons number between 0 and 32.
name - "16 ASCII Chars"

**Comment by catch-point** [comment]
Similar behaviour (camera crashes) when attempting to switch to a custom preset

**Reply by dnewman-gpsw** [reply]
Already fixed, out next Tuesday.
